core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -0,0 +1,755 @@
+package com.scalar.db.api;
+
+import com.scalar.db.common.error.CoreError;
+import com.scalar.db.exception.storage.ExecutionException;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+
+/** An interface for administrative operations for Attribute-Based Access Control. */
```

Comment:
Since the error message refers to `attribute-based access control` in lowercase, can we lowercase it in the code comments as well for consistency? That way, we won't have to wonder if the uppercase or lowercase spelling is accurate when we reference this in the future.